client/schema.go

```diff
@@ -94,27 +94,20 @@ func NewHttpSchemaFetcher(httpClient http.Client, address string, scope tally.Sc
 }
 
 // Start starts the CachedSchemaHandler, if interval > 0, will start periodical refresh
-func (cf *CachedSchemaHandler) Start(interval int) error {
-	err := cf.FetchAllSchema()
-	if err != nil {
-		return err
-	}
+func (cf *CachedSchemaHandler) Start(interval int) {
+	cf.FetchSchemas()
```

Comment:
isn't this always a no-op?
cf.schemas will start with empty map, only Insert() calls will start fetching schema lazily.

Reply:
Yes, this is my intention to make ares-subscriber initialization fast. Fetch schema and enum is by demand.

Reply:
yeah then we can remove this call from the code.

Reply:
I keep this b/c in case the other features will set tables in cachedSchemaHandler during initialization in the future. Then Start() will not delay the fetch in that case. And also in the current use case, the cost of this call can be ignored.